staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/naming_controller.go

```diff
@@ -191,26 +191,26 @@ func (c *NamingConditionController) calculateNamesAndConditions(in *apiextension
 		namesAcceptedCondition.Message = "no conflicts found"
 	}
 
-	// set EstablishedCondition to true if all names are accepted. Never set it back to false.
-	establishedCondition := apiextensions.CustomResourceDefinitionCondition{
-		Type:    apiextensions.Established,
+	// set InstallingCondition to true if all names are accepted. Never set it back to false.
```

Comment:
"Never set it back" is strange now. Maybe "Never set it back here to false (the established controller will do)"